tests/w2w_test.py

```diff
@@ -877,7 +887,14 @@ def test_create_lcz_params_file_attrs_type(
     assert nbui_max == 5
 
 
-def test_create_lcz_extent_file(tmpdir, info_mock):
+@pytest.mark.parametrize(
+    ('version', 'num_land_cat'),
+    (
+        ('v4.3', 41),
+        ('v4.5', 61),
+    ),
+)
+def test_create_lcz_extent_file(tmpdir, info_mock, version, num_land_cat):
```

Comment:
This kinda tests that the version triggers different numbers

Reply:
Thanks!